tests/fuzzy/test_orders_fuzzy.py

```diff
@@ -17,45 +17,45 @@
 @mark.parametrize('orderType,numOrders,withBoundingOrders,deadOrderProbability', [
     (BID,  10, False,  0.0),
     (ASK,  10, False,  0.0),
-    (BID,  50, False,  0.0),
-    (ASK,  50, False,  0.0),
-    (BID, 100, False,  0.0),
-    (ASK, 100, False,  0.0),
+    (BID,  20, False,  0.0),
```

Comment:
I picked these since it's at least doubling instead of totally arbitrary and because this makes the fuzzy test speed run on part with the reporting suite so it isn't the bottleneck in CI